client/lib/credit-card-details/validation.js

```diff
@@ -5,50 +5,100 @@
  */
 
 import creditcards from 'creditcards';
-import { capitalize, compact, inRange, isArray, isEmpty } from 'lodash';
+import capitalize from 'lodash/capitalize';
+import compact from 'lodash/compact';
+import inRange from 'lodash/inRange';
+import isArray from 'lodash/isArray';
+import isEmpty from 'lodash/isEmpty';
+import pick from 'lodash/pick';
 import i18n from 'i18n-calypso';
 
-function creditCardFieldRules() {
-	return {
-		name: {
-			description: i18n.translate( 'Name on Card', {
-				context: 'Upgrades: Card holder name label on credit card form',
-				textOnly: true,
-			} ),
-			rules: [ 'required' ],
-		},
+/**
+ * Internal dependencies
+ */
+import { isEbanx, isValidCPF } from 'lib/credit-card-details/ebanx';
+import { PAYMENT_PROCESSOR_EBANX_COUNTRIES } from './constants';
 
-		number: {
-			description: i18n.translate( 'Card Number', {
-				context: 'Upgrades: Card number label on credit card form',
-				textOnly: true,
-			} ),
-			rules: [ 'required', 'validCreditCardNumber' ],
-		},
+function ebanxFieldRules( country ) {
+	const requiredFields = PAYMENT_PROCESSOR_EBANX_COUNTRIES[ country ].requiredFields || [];
 
-		'expiration-date': {
-			description: i18n.translate( 'Credit Card Expiration Date' ),
-			rules: [ 'required', 'validExpirationDate' ],
-		},
+	return pick(
+		{
+			document: {
+				description: i18n.translate( 'Taxpayer Identification Number' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Taxpayer Identification Number (TIN)' )` *ES Score: 28*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).